README.md

```diff
@@ -107,6 +107,8 @@ The following environment variables are available, and all of them are optional.
 
 #### `ENABLE_V6_EGRESS` (v1.13.0+)
 
+🚨 This feature is under active development. This feature is not released. 
```

Comment:
`v1.13.0` has not been released yes, so I think this is inferred. We have not used notes like this for any other features, so I think this is not needed